db/migrate/20130409133705_create_timelines_alternate_dates.rb

```diff
@@ -39,7 +39,7 @@ def self.up
       t.timestamps
 
     end
-    add_index :timelines_alternate_dates, :planning_element_id
+    add_index :timelines_alternate_dates, :planning_element_id, :name => "idx_tadpei"
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Prefer single-quoted strings when you don't need string interpolation or special symbols.